docs/getting-started/advanced-setting.asciidoc

```diff
@@ -67,6 +67,16 @@ IMPORTANT: {es-sec} requires {ecs-ref}[ECS-compliant data]. If you use third-par
 collectors to ship data to {es}, the data must be mapped to ECS.
 <<siem-field-reference>> lists ECS fields used in {es-sec}.
 
+[discrete]
+[[update-threat-intel-indices]]
+== Update default Elastic Security threat intelligence indices
+
+The `securitySolution:defaultThreatIndex` advanced setting defines threat intelligence indices that {elastic-sec} will use when collecting threat match data. The setting controls features that query threat indices, such as the Threat Intelligence view on the Overview page and threat indicator match rules. One or more threat intelligence indices can be defined; the `filebeat-*` index is specified by default.
```

Comment:
we could consider saying `collecting indicators` instead of `collect threat match data`, as the matching happens at a later stage